modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/H2DynamicTableSelfTest.java

```diff
@@ -619,6 +621,29 @@ public void testQueryParallelism() {
         assertQueryParallelism("QP_OVERWRITE_TEMPLATE", 42);
     }
 
+    /**
+     * Test keywords in table name.
+     * Try to create table with all keywords.
+     */
+    @Test
+    public void testCreateTableWithKeywordName() {
+        int i = 0;
+
+        Set<String> keywords = SqlKeyword.getKeywords();
+
+        for (String entry : keywords
```

Comment:
fix here + after this issue we can`t call "create table help" and so on ? Is it ok ? + do you really need keywords as public method ?

Reply:
1) HELP - If this word is in keywords why should it be allowed to be used?
2) public - I can agree because at the moment it is used only in this test, for this i can go in another way. But why not, it seems that this is the simplest solution?

p.s. my english is bad and i use google translate.

Reply:
Sasha, Zhenya meant that you didn't have to implement the public method `getKeywords()` just for testing purposes.
You should use `U.staticField(SqlKeyword.class, "KEYWORDS")`

Reply:
@zstan If one wants to create a table with name `help`, it is still possible to surround the name with quotes.

Reply:
2. Ok - I'll fix it